subsys/net/lib/nrf_cloud/coap/src/nrf_cloud_coap_transport.c

```diff
@@ -337,7 +384,7 @@ static int client_transfer(enum coap_method method,
 			   enum coap_content_format fmt_in,
 			   bool response_expected,
 			   bool reliable,
-			   struct cc_xfer_data *const xfer)
+			   struct cc_xfer_data *xfer)
 {
 	__ASSERT_NO_MSG(resource != NULL);
 	__ASSERT_NO_MSG(xfer != NULL);
```

Comment:
this should be changed to an `if (xfer == NULL)` check, since now there is a chance that xfer could be NULL.

Reply:
Ugh. You asked me to remove all the other checks below it for whether xfer == NULL.